core/src/test/scala/unit/kafka/controller/ControllerIntegrationTest.scala

```diff
@@ -380,12 +471,13 @@ class ControllerIntegrationTest extends ZooKeeperTestHarness {
       leaderIsrAndControllerEpoch.leaderAndIsr.leader == leader &&
       leaderIsrAndControllerEpoch.leaderAndIsr.leaderEpoch == leaderEpoch
 
-  private def makeServers(numConfigs: Int, autoLeaderRebalanceEnable: Boolean = false, uncleanLeaderElectionEnable: Boolean = false) = {
+  private def makeServers(numConfigs: Int, autoLeaderRebalanceEnable: Boolean = false, uncleanLeaderElectionEnable: Boolean = false, deleteTopicEnable: Boolean = false) = {
     val configs = TestUtils.createBrokerConfigs(numConfigs, zkConnect)
     configs.foreach { config =>
       config.setProperty(KafkaConfig.AutoLeaderRebalanceEnableProp, autoLeaderRebalanceEnable.toString)
       config.setProperty(KafkaConfig.UncleanLeaderElectionEnableProp, uncleanLeaderElectionEnable.toString)
       config.setProperty(KafkaConfig.LeaderImbalanceCheckIntervalSecondsProp, "1")
+      config.setProperty(KafkaConfig.DeleteTopicEnableProp, deleteTopicEnable.toString)
```

Comment:
Do we need ```deleteTopicEnable: Boolean = false``` flag?  looks like none of the tests depends on deleteTopicEnable=false. looks like all tests arel passing without this line

Reply:
Yes. I added that for `testControllerMoveOnTopicDeletion` but it turns out we don't actually need to enable topic deletion to test throwing and handling `ControllerMovedException` happening in `TopicDeletion` event. 
Fixed.